CONTRIBUTING.quickstart.md

```diff
@@ -0,0 +1,294 @@
+# Cypress Contributor Quick Start Guide
+
+Welcome! This guide gets you coding on Cypress in under 15 minutes. Skip the overwhelming details and focus on the essentials to start contributing effectively.
+
+## 🚀 Quick Setup (5 minutes)
+
+### Prerequisites
+```bash
+# Check your Node.js version (required: ^20.1.0 || ^22.0.0 || >=24.0.0)
+node --version
+
+# If wrong version, use nvm
+nvm install 22.0.0
+nvm use 22.0.0
```

Comment:
`nvm` is not installed by default. It is one of several optional Node.js version managers. Instructions to install it are missing. There are different versions of `nvm`, depending on the operating system (Linux, macOS or Windows).